src/io/hdf/hdf_stl.h

```diff
@@ -144,6 +144,82 @@ struct h5data_proxy<std::string>
   }
 };
 
+// Specialization for vector of strings
+template<>
+struct h5data_proxy<std::vector<std::string>>
```

Comment:
New code should conform to the current coding conventions.
i.e.
using instead of type Alias
TypeName instead of TypeName_t